src/Catalyst.Cli.Tests/UnitTests/Commands/Response/GetVersionResponseTests.cs

```diff
@@ -42,7 +42,7 @@ public void GetVersionResponse_Can_Get_Output()
             //Arrange
             var versionResponse = new VersionResponse {Version = "1.2.3.4"};
             var commandContext = TestCommandHelpers.GenerateCliResponseCommandContext(_testScheduler);
-            var getVersionCommand = new GetVersionCommand(commandContext, Substitute.For<ILogger>());
+            new GetVersionCommand(commandContext, Substitute.For<ILogger>());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove this useless object instantiation of class 'GetVersionCommand' or use it.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)